aspnetcore/fundamentals/configuration/index.md

```diff
@@ -1023,6 +1072,8 @@ In the sample app, a POCO class is available to hold the bound configuration dat
 The configuration data is bound to the object:
 
 ```csharp
+// using Microsoft.Extensions.Configuration;
```

Comment:
This is repeated throughout -- you might consider including it only the first one or two times and omitting it from the rest.